### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by sirojan24**
Fix the review comments

### CODE REVIEW

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp [hunk 1]
```diff
@@ -129,8 +129,7 @@
                 CarbonUIMessage.sendCarbonUIMessage(e.getMessage(), CarbonUIMessage.ERROR, request, e);
```
Comment (sajinidesilva) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp]: Need to remove this line also.

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp [hunk 2]
```diff
@@ -299,10 +298,9 @@
         e.printStackTrace();
 %>
 
-<script type="text/javascript">
-    location.href = "../admin/error.jsp";
-    alert("error");
-</script>
+    <script type="text/javascript">
+        CARBON.showErrorDialog('<%=e.getFaultMessage().getBrokerManagerAdminException().getErrorMessage()%>');
```
Comment (sajinidesilva) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp]: This is wrong. e.getFaultMessage is not defined. Should be e.getMessage()

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp (hunk 1 above)
Comment (sirojan24) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp]: Fixed

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp (hunk 2 above)
Comment (sirojan24) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp]: Fixed

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp [hunk 3]
```diff
@@ -119,18 +119,16 @@
...
                     allClusterNodeAddressesInDropdown = (String[]) clusterNodesDropdownList.toArray(new String[0]);
                 }
                 nodeId = client.getMyNodeID();
             } catch (Exception e) {
-                CarbonUIMessage.sendCarbonUIMessage(e.getMessage(), CarbonUIMessage.ERROR, request, e);
             %>
             <script type="text/javascript">
-                location.href = "../admin/error.jsp";
-                alert("error");
+                CARBON.showErrorDialog('<%=e.getMessage()%>');
```
Comment (sajinidesilva) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp]: Its better if we can add the log with a customize message.

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp [hunk 4]
```diff
@@ -174,20 +172,16 @@
...
             }
         } catch (Exception e) {
-            CarbonUIMessage.sendCarbonUIMessage(e.getMessage(), CarbonUIMessage.ERROR, request, e);
-            e.printStackTrace();
     %>
-
     <script type="text/javascript">
-        location.href = "../admin/error.jsp";
-        alert("error");
+        CARBON.showErrorDialog('<%=e.getMessage()%>');
```
Comment (sajinidesilva) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp]: Please show the error with a customized error message.

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp [hunk 5]
```diff
@@ -179,18 +179,16 @@
...
            allClusterNodeAddressesInDropdown = (String[]) clusterNodesDropdownList.toArray(new String[0]);
         }
         nodeId = client.getMyNodeID();
     } catch (Exception e) {
-        CarbonUIMessage.sendCarbonUIMessage(e.getMessage(), CarbonUIMessage.ERROR, request, e);
     %>
     <script type="text/javascript">
-        location.href = "../admin/error.jsp";
-        alert("error");
+        CARBON.showErrorDialog('<%=e.getMessage()%>');
```
Comment (sajinidesilva) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp]: Good to have a customized error message.

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp [hunk 6]
```diff
@@ -295,14 +293,10 @@
...
-        CarbonUIMessage.sendCarbonUIMessage(e.getMessage(), CarbonUIMessage.ERROR, request, e);
-        e.printStackTrace();
 %>
-
-<script type="text/javascript">
-    location.href = "../admin/error.jsp";
-    alert("error");
-</script>
+    <script type="text/javascript">
+        CARBON.showErrorDialog('<%=e.getMessage()%>');
```
Comment (sajinidesilva) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp]: Good to have a customized error message.

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp (hunk 4 above)
Comment (sirojan24) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/queue_subscriptions_list.jsp]: Customized error message added in the latest commit

File: components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp (hunk 5 above)
Comment (sirojan24) [inline components/andes/org.wso2.carbon.andes.ui/src/main/resources/web/subscriptions/topic_subscriptions_list.jsp]: Customized error message added in the latest commit

**Pull request merged**